lib/elixir_internal_certificate/scrapper/schemas/search_result.ex

```diff
@@ -9,13 +13,24 @@ defmodule ElixirInternalCertificate.Scrapper.Schemas.SearchResult do
     field :preview, :string
 
     belongs_to :user_search,
-               ElixirInternalCertificate.Scrapper.Schemas.UserSearch,
+               UserSearch,
                foreign_key: :user_search_id
 
     has_many :url_results,
-             ElixirInternalCertificate.Scrapper.Schemas.UrlResult,
+             UrlResult,
              foreign_key: :search_result_id
 
     timestamps()
   end
+
+  def search_result_changeset(search_result \\ %__MODULE__{}, attrs),
+    do:
```

Comment:
should be do end block

Reply:
@junan Sorry I kind of don't get your point. The above block is a 1 line function without any pipeline, why should it write as a do-end block.

Reply:
> keep the function body just in one line

Here "function body" means the content after the `do`

![Screen Shot 2565-06-29 at 3 19 00 PM](https://user-images.githubusercontent.com/6076762/176387871-cbf7b838-b997-4725-a0f9-bc444066c558.png)

Reply:
@junan so it would be single-line functions if and only the body has 1 line only right?

Reply:
yes